### PR TITLE
feat: add timeframe to nodecleaner service

### LINKED ISSUE #63: Add timeframe to nodecleaner service
`services/cronjob/nodecleaner/internal/service/node_service.go` wipes all `validation_failed` status node profiles at a predetermined interval (currently every 5 minutes).

There is an issue here if a user creates a profile that fails validation right before the cleaning interval is reached. It will be there in the DB one minute and then disappear the next. This behavior is unusual on the front end because the user sees the invalid profile in their profile list and then it disappears. Therefore, let's introduce a configurable timeframe such that any invalid profiles that are newer than that timeframe will not be cleaned from the DB.

Also, please ensure that the cronjob time is configurable based on environment (staging or production).

**Staging**
- run nodecleaner every 2 minutes
- only delete profiles if they were created more than 4 minutes ago

**Production**
- run nodecleaner every 12 hrs
- only delete profiles if they were created more than 24 hours ago

### PR DESCRIPTION
**Pull request by maxlivinci**
resolves #63

### CODE REVIEW

File: charts/murmurations/charts/nodecleaner/templates/nodecleaner/cronjob.yaml [hunk 1]
```diff
@@ -3,7 +3,13 @@ kind: CronJob
 metadata:
   name: nodecleaner
 spec:
-  schedule: "*/5 * * * *"
+  {{- if eq .Values.global.env "production" }}
```
Comment (maxlivinci) [inline charts/murmurations/charts/nodecleaner/templates/nodecleaner/cronjob.yaml:6]: cronjob time

File: charts/murmurations/charts/nodecleaner/templates/nodecleaner/config.yaml [hunk 2]
```diff
@@ -5,3 +5,10 @@ metadata:
 data:
   INDEX_MONGO_URL: mongodb://index-mongo:27017
   INDEX_MONGO_DB_NAME: murmurationsIndex
+  {{- if eq .Values.global.env "production" }}
```
Comment (maxlivinci) [inline charts/murmurations/charts/nodecleaner/templates/nodecleaner/config.yaml:8]: configurable timeframe

File: charts/murmurations/charts/nodecleaner/templates/nodecleaner/config.yaml [hunk 3]
```diff
@@ -5,3 +5,10 @@ metadata:
...
+  TTL: "43200" # 12 hrs
```
Comment (maxlivinci) [inline charts/murmurations/charts/nodecleaner/templates/nodecleaner/config.yaml:9]: configurable timeframe

File: charts/murmurations/charts/nodecleaner/templates/nodecleaner/cronjob.yaml [hunk 4]
```diff
@@ -3,7 +3,13 @@ kind: CronJob
...
+  schedule: "0 */12 * * *" # every 12 hrs
```
Comment (maxlivinci) [inline charts/murmurations/charts/nodecleaner/templates/nodecleaner/cronjob.yaml:7]: configurable cronjob time

**Review by geoffturk — APPROVED** [review]
LGTM 🚀

**Pull request merged**